tests/README.md

```diff
@@ -0,0 +1,67 @@
+# HubSpot CLI E2E tests
+
+To assist in adding test coverage to the CLI, this project attempts to test `hs.*` commands as if they were being used by an end-user.
+
+## Getting Started
+
+The main test execution is kicked of via the `run-tests` script. `run-tests` itself is a CLI, so to see the available options, you can `run-tests --help`. This was done so that the tests can be installed against any given version of teh CLI and tested against it. In addition, it was done to keep the test logic separated from the CLI library
```

Comment:
A few typos in here. "kicked off*", "version of the* CLI", and missing period at the end.